### DISCUSSION TITLE
How to import feeds from OPML files?

### DISCUSSION THREAD

**Discussion by Tony20221** [martinrotter/rssguard#1435] [Q&A]
How to import feeds from an OPML file? I can't find any import option in the feeds section.

**Comment by martinrotter** [comment]
go to account menu -> your active rss "account" or right click your "account" node in feed list

![rssguard_GBvcmE9Jh0](https://github.com/martinrotter/rssguard/assets/1255302/21983c6f-84c7-4995-b7c1-9cc5a06b95e2)

**Comment by Korb** [comment]
> How to import feeds from an OPML file? I can't find any import option in the feeds section.

Or right-click on the account in the sidebar and select `Import feeds`:

![firefox (2025-06-14 12-37-13)](https://github.com/user-attachments/assets/ee93d975-6f55-447f-a9b5-4fa8c15a5d95)
